content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,147 @@
+
+| Language | Minimum version required |
+|----------|--------------------------|
+| Java     | v1.35.0                  |
+| Go       |                          |
+| Python   |                          |
+| Ruby     |                          |
+| Node.js  |                          |
+| PHP      |                          |
+| .NET     |                          |
+
+{{< tabs >}}
+
+{{% tab "Java" %}}
+
+## Compatibility requirements
+
+
+
+Each instrumentation should be packaged as an OpenTelemetry [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/README.md).
+The SDK also accepts selected individual instrumentation jars produced by OpenTelemetry's [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main)
+build, for example the [CFX instrumentation jar](https://search.maven.org/search?q=a:opentelemetry-javaagent-jaxws-2.0-cxf-3.0).
+
+**Note**: Use of OpenTelemetry incubator APIs is not currently supported.
+
+## Getting started
+
+To use an OpenTelemetry instrumentation with the Datadog SDK for Java:
+1. Set the `dd.trace.otel.enabled` system property or the `DD_TRACE_OTEL_ENABLED` environment variable to true
+2. Download (or build) the OpenTelemetry instrumentation as an extension jar
+3. Set the `otel.javaagent.extensions` system property or the `OTEL_JAVAAGENT_EXTENSIONS` environment variable to
+   the path to the extension jar
+
+## Configuration
+
+All configuration options below have system property and environment variable equivalents.
+If the same key type is set for both, the system property configuration takes priority.
+System properties should be set as JVM flags.
+
+`dd.trace.otel.enabled`
+: **Environment Variable**: `DD_TRACE_OTEL_ENABLED`<br>
+**Default**: `false`<br>
+Must be set to `true` to enable use of OpenTelemetry instrumentations.
+
+`otel.javaagent.extensions`
+: **Environment Variable**: `OTEL_JAVAAGENT_EXTENSIONS`<br>
+**Default**: `false`<br>
+A comma-separated list of paths to extension jar files, or folders containing extension jar files.
+
+OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/java/agent/configuration/) page describes additional properties which are also recognized by the Datadog SDK.
+
+## Verified OpenTelemetry Extensions
+
+| Framework           | Versions | OpenTelemetry Extension                                                                                                    | Instrumentation Names |
+|---------------------|----------|----------------------------------------------------------------------------------------------------------------------------|-----------------------|
+| Apache CXF (Jax-WS) | 3.0+     | [opentelemetry-javaagent-jaxws-2.0-cxf-3.0](https://search.maven.org/search?q=a:opentelemetry-javaagent-jaxws-2.0-cxf-3.0) | `cxf`                 |
+
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+## Compatibility requirements
```

Comment:
Are these going to be filled in as part of this PR? If not, consider removing these or commenting them out for now

Reply:
These are common placeholders for the different language experts to fill in - if they decide a section isn't required/applicable to their language then it could be removed, but I would like to keep some consistency across the tags where possible.

Reply:
I've commented out the empty tabs for now and put `TBA` placeholders in the table - that way we can merge this once we're happy with the general and Java specific content (each language can then work on their own PR)